lib/Model/File.php

```diff
@@ -59,6 +62,20 @@ public function setType($type) {
 		return $this;
 	}
 
+	/**
+	 * @return mixed
+	 */
```

Comment:
why mixed?

Reply:
@micbar Our api returns string on some databases while int is returned on others